lib/ae_mdw/auction_bids.ex

```diff
@@ -12,27 +12,70 @@ defmodule AeMdw.AuctionBids do
 
   @table Model.AuctionBid
 
+  @type cursor :: binary()
+  # This needs to be an actual type like AeMdw.Db.Name.t()
+  @type auction_bid() :: term()
+
+  @typep order_by :: :expiration | :name
+  @typep limit :: Mnesia.limit()
+  @typep direction :: Mnesia.direction()
   @typep plain_name() :: binary()
-  @typep auction_bid() :: term()
 
-  @spec top_auction_bid(plain_name()) :: {:ok, auction_bid()} | :not_found
-  def top_auction_bid(plain_name) do
+  @table Model.AuctionBid
+  @table_expiration Model.AuctionExpiration
+
+  @spec top_auction_bid(plain_name(), boolean()) :: {:ok, auction_bid()} | :not_found
+  def top_auction_bid(plain_name, expand?) do
     case Mnesia.prev_key(@table, bid_top_key(plain_name)) do
       {:ok, auction_bid} ->
-        if(elem(auction_bid, 0) == plain_name, do: {:ok, render(auction_bid)}, else: :not_found)
+        if(elem(auction_bid, 0) == plain_name,
+          do: {:ok, render(auction_bid, expand?)},
+          else: :not_found
+        )
```

Comment:
nitpick to if do else without inliner `, do:`

Reply:
Good call. Done